Monitron.Common/TestPlugin.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Monitron.Common
+{
+	public sealed class TestPlugin: IPlugin
+	{
+		IMessengerClient m_MessangerClient;
+		public IMessengerClient MessangerClient
+		{
+			get {return m_MessangerClient;}
+		}

```

Comment:
Could just use autmatic property with a private setter instead of defining a member.

https://msdn.microsoft.com/en-us/library/bb384054.aspx